### DISCUSSION TITLE
[FEATURE] Adding Thumb Up/Thumb Down (Rating) Feature to NPM website for various packages

### DISCUSSION THREAD

**Discussion by jbactual** [npm/rfcs#98] [General]
## Problem/Issue
* Downloading multiple npm packages just to find out each one is inadequate for what you need.  

## What/Why
* What: Add some type of rating system for all the various packages on npm.  Maybe allow users to comment on packages as well.

* Why: Allows users to weed out the good and bad packages.  Allows NPM governing body to eliminate bad or non-used packages.  

## Where
* npm website / database

**Comment by wesleytodd** [comment]
A few thoughts here:

1. I am not sure this is the right place for this type of request.  This repo is for CLI, not the website
2. I am a 👎 on social features in the registry.  It is hard enough for package maintainers and adding more things they need to worry about in this way is not a good thing.  Also, I don't think this kind of signal is what will help you find "adequate" packages.
3. There is no governing body like this. There are processes for malicious packages and non-used packages changing hands, but I don't personally want it based on user ratings.

**Comment by vreemt** [comment]
I feel the current stats (version history, weekly downloads etc) and readme of a package are generally sufficient to provide usability details

Off-topic: Where is the right place to let NPM know about website features? The contact form mentions an e-mail address, but I feel that's possibly not the most efficient way. 
eg. my suggestion @ https://github.com/npm/cli/issues/836#issuecomment-586993798_
